examples/platform/efr32/efr32_utils.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2022 Silabs.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+// EFR Logging
+#ifdef __cplusplus
+extern "C" {
```

Comment:
needs `#pragma once`.

Could we avoid names like `util`, `helpers`, `support` ? this seems to define logging bits. How about some `matter_efr32_logging` or similar?

Reply:
Fixed, will renamed the file in the near future, as we plan on doing more cleanups/upgrade of our examples.